src/statics/voidAll.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/**
+ * Like Promise.all, but without examining or collecting the resolved values from the Promises.
+ */
+function voidAll(iter) {
+  return new Promise((resolve, reject) => {
+    let n = 0,
+      total = 0,
+      ready = false;
+
+    try {
+      for (const value of iter) {
+        Promise.resolve(value).then(() => {
+          ++n;
+          if (ready && n === total) resolve();
```

Comment:
The `ready` variable (and check) shouldn't strictly speaking be necessary due to the way Promises work, but it feels like a useful safety belt in case the environment is using a non-standard implementation - at least to expose the problem instead of masking it.

Reply:
For future reference, this behavior is defined:

https://tc39.es/ecma262/#sec-performpromisethen item 9
https://tc39.es/ecma262/#sec-triggerpromisereactions